strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFileAttributes.java

```diff
@@ -0,0 +1,154 @@
+package org.carlspring.strongbox.providers.io;
+
+import java.nio.file.attribute.BasicFileAttributes;
+import java.nio.file.attribute.FileTime;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.carlspring.strongbox.artifact.coordinates.ArtifactCoordinates;
+
+public class RepositoryFileAttributes implements BasicFileAttributes
+{
+
+    public static final String COORDINATES = "coordinates";
+    public static final String IS_METEDATA = "isMetedata";
```

Comment:
First, there's a typo here -- "isMet**e**data", second of all can these fields not be called "METADATA", "CHECKSUM", etc? Why do they need to have `boolean` names?